pylint_odoo/examples/.jslintrc

```diff
@@ -285,5 +285,7 @@
     "yield-star-spacing": "off",
     "yoda": "error"
   },
-  "parserOptions": {}
+  "parserOptions": {
+    "ecmaVersion": 6,
```

Comment:
- Odoo 13 uses [`async/await` in normal code](https://github.com/odoo/odoo/blob/4ad7a99df0a25122cf6872f880e07b6148c82ea8/addons/web/static/src/js/views/abstract_controller.js#L206) (outside of tests), which is ES2016 (ES7)
- [Also uses the spread operator for browser code](https://github.com/odoo/odoo/blob/4ad7a99df0a25122cf6872f880e07b6148c82ea8/addons/website/static/src/js/content/snippets.animation.js#L693), which is ES2018 (ES9).

So, this should be:

```suggestion
    "ecmaVersion": 2018,
```

(It seems to me that year-versioning is easier to understand these days).

---

Note: [Year/version/features summary](https://en.wikipedia.org/wiki/ECMAScript).